src/kernel/include/shared/cos_types.h

```diff
@@ -840,8 +848,8 @@ static inline void
 cos_mem_fence(void)
 { __asm__ __volatile__("mfence" ::: "memory"); }
 
-// ncpu * 16 (or max 256) entries. can be increased if necessary.
-#define COS_THD_INIT_REGION_SIZE (((NUM_CPU*16) > (1<<8)) ? (1<<8) : (NUM_CPU*16))
+/* 256 entries. can be increased if necessary */
+#define COS_THD_INIT_REGION_SIZE (1<<8)
```

Comment:
Yes, I've changed it to 256. 
I meant I didn't have a valid argument to removing NUM_CPUs, I only could have changed the const from 16 -> 32/64. 

Thank you.